src/main/java/io/fabric8/jenkins/openshiftsync/BuildWatcher.java

```diff
@@ -197,48 +195,23 @@ private static synchronized void buildModified(Build build) {
     }
   }
 
-  public static synchronized void buildAdded(Build build) throws IOException {
+  public static synchronized boolean buildAdded(Build build) throws IOException {
```

Comment:
In `triggerJob` (https://github.com/jenkinsci/openshift-sync-plugin/pull/26/files#diff-30f5e3ec15ac23f3c5ef1ea07725159bR212)